libs/ledgerjs/packages/cryptoassets/src/data/filecoin-erc20.json

```diff
@@ -0,0 +1,98 @@
+[
```

Comment:
We are now using "importers" now to dynamically fetch token info from our CDN from backend, instead of hard code from in the codebase.

Please refer to:
/ledger-live/libs/ledgerjs/packages/cryptoassets/src/crypto-assets-importer/importers

We are currently working with our backend team to host filecoin erc20 token. We will let you know when the CDN is ready, so that you can do that in dynamic/ importer way.

Reply:
Great thanks @hzheng-ledger! Now using importer for fetching tokens.